Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1174,7 +1200,7 @@ theorem ne_sup_iff_lt_sup {ι : Type u} {f : ι → Ordinal.{max u v}} :
   ne_iSup_iff_lt_iSup
 
 -- TODO: state in terms of `IsSuccLimit`.
-theorem succ_lt_iSup_of_ne_iSup {ι : Type u} {f : ι → Ordinal.{max u v}}
+theorem succ_lt_iSup_of_ne_iSup {ι} {f : ι → Ordinal.{u}} [Small.{u} ι]
```

Comment:
This and below should also be generalized and possibly deprecated at some point, but that can happen later, and this improvement is good in the meanwhile.